tasks/certbot.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Prepare cron for autocert bot
+  apt: name=cron state=present
+
+- name: Stop "{{ jellyfin_HTTP_server }}" to prepare for certbot challenge
+  systemd:
+    name: "{{ jellyfin_HTTP_server }}"
+    state: stopped
```

Comment:
Using the webroot method instead of the standalone method, there is no need to stop nginx, only to reload its configuration after the certificate is created.